tools/Tools.Common/SerializedCmdlets/Az.Network.json

```diff
@@ -86799,7 +86799,8 @@
             "BeginsWith",
             "EndsWith",
             "Regex",
-            "GeoMatch"
+            "GeoMatch",
+            "Any"
```

Comment:
No need to update this file. This file will be updated automatedly when we build the release.